README.md

```diff
@@ -20,24 +20,36 @@ cd $GOPATH/src/github.com/lightninglabs/faraday
 make && make install
 ```
 
-#### Tests
-To run all the unit tests in the repo, use:
-
-```
-make check
-```
-
 ## Usage
 Faraday connects to a single instance of lnd. It requires access to macaroons for each subserver and a valid TLS certificate. It will attempt to use the default lnd values if no command line flags are specified.
 ```
-./faraday                                    \
+./faraday                                       \
 --macaroondir={directory containing macaroon}   \
 --tlscertpath={path to lnd cert}                \
---rpserver={host:port of lnd's rpserver} 
+--rpcserver={host:port of lnd's rpcserver} 
 ```
 
 By default, faraday runs on mainnet. The `--testnet`, `--simnet` or `--regtest` flags can be used to run in test environments.
 
+### Chain Backend
+Faraday offers node accounting services which require access to a Bitcoin node with `--txindex` set so that it can perform transaction lookup. Currently the `NodeReport` and `CloseReport` endpoints require this connection, and will fail if it is not present. This connection is *optional*, and all other endpoints will function if it is not configured. 
+
+To connect Faraday to bitcoind:
+```
+--connect_bitcoin                       \
+--bitcoin.host={host:port of bitcoind}  \
+--bitcoin.user={bitcoind username}      \
+--bitcoin.password={bitcoind  password}
```

Comment:
I think you need to set `--httppostmode` for `bitcoind`.

Reply:
We default to true here, which is cheeky, but btcd just ignores it so I think it's ok.